packages/ui/src/utils/renderWithTheme.ts

```diff
@@ -0,0 +1,9 @@
+import { ReactElement } from 'react';
```

Comment:
💭 For consistency with [another PR](https://github.com/CodersCrew/coderscamp/pull/126) add this function to `testHelpers.ts` file in the `src` directory